samples/peripherals/buttons/prj.conf

```diff
@@ -1,6 +1,9 @@
 CONFIG_LOG=y
 CONFIG_LOG_BACKEND_BM_UARTE=y
 
```

Comment:
I was wondering, maybe a short comment to indicate the reason for adding it and prevent confusion?